cluster/saltbase/salt/kubelet/default

```diff
@@ -15,6 +15,15 @@
   {% set api_servers = "--api_servers=https://" + ips[0][0] + ":6443" -%}
 {% endif -%}
 
+# Disable registration for the kubelet running on the master on GCE.
```

Comment:
Fine with you doing the master registration in a future PR, as long as it gets done at some point (with appropriate adjustments for GKE)